lib/main.js

```diff
@@ -407,16 +407,21 @@ function initSubscriptions() {
       minimapElement.attach(textEditor.getElement())
     }),
     // empty color cache if the theme changes
-    atom.themes.onDidChangeActiveThemes(() => {
-      domStylesReader.invalidateDOMStylesCache()
-      editorsMinimaps.forEach((minimap) => {
-        atom.views.getView(minimap).requestForcedUpdate()
-      })
-    }),
+    atom.themes.onDidChangeActiveThemes(updateStyles),
+    atom.styles.onDidUpdateStyleElement(updateStyles),
+    atom.styles.onDidAddStyleElement(updateStyles),
+    atom.styles.onDidRemoveStyleElement(updateStyles),
     treeSitterWarning()
   )
 }
 
+function updateStyles() {
+  domStylesReader.invalidateDOMStylesCache()
+  editorsMinimaps.forEach((minimap) => {
+    atom.views.getView(minimap).requestForcedUpdate()
+  })
+}
```

Comment:
We might want to debounce this function since it looks like Atom calls it multiple times when styles.less is changed.

Reply:
I see we have underscore-plus as a dependency so this should be easy.